dashboard/src/pages/TreeDetails/Tabs/Boots/BootsTable.tsx

```diff
@@ -0,0 +1,294 @@
+        ),
+        value: possibleTestsTableFilter[1],
+        isSelected: checkIfFilterIsSelected(possibleTestsTableFilter[1]),
+      },
+      {
+        label: intl.formatMessage(
+          { id: 'global.failedCount' },
+          { count: filterCount[possibleTestsTableFilter[2]] },
+        ),
+        value: possibleTestsTableFilter[2],
+        isSelected: checkIfFilterIsSelected(possibleTestsTableFilter[2]),
+      },
+      {
+        label: intl.formatMessage(
+          { id: 'global.inconclusiveCount' },
+          { count: filterCount[possibleTestsTableFilter[3]] },
+        ),
+        value: possibleTestsTableFilter[3],
+        isSelected: checkIfFilterIsSelected(possibleTestsTableFilter[3]),
+      },
+    ],
+    [intl, filterCount, checkIfFilterIsSelected],
+  );
+
+  const table = useReactTable({
+    data,
+    columns,
+    onSortingChange: setSorting,
+    getCoreRowModel: getCoreRowModel(),
+    getPaginationRowModel: getPaginationRowModel(),
+    onPaginationChange: setPagination,
+    getSortedRowModel: getSortedRowModel(),
+    getFilteredRowModel: getFilteredRowModel(),
+    state: {
+      sorting,
+      pagination,
+    },
+  });
+
+  const groupHeaders = table.getHeaderGroups()[0].headers;
+  const tableHeaders = useMemo((): JSX.Element[] => {
+    return groupHeaders.map(header => {
+      return (
+        <TableHead key={header.id}>
+          {header.isPlaceholder
+            ? null
+            : flexRender(header.column.columnDef.header, header.getContext())}
+        </TableHead>
+      );
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [groupHeaders, sorting]);
+
+  const modelRows = table.getRowModel().rows;
+  const tableRows = useMemo((): JSX.Element[] => {
+    return modelRows?.length
+      ? modelRows.map(row => (
+          <TableRow key={row.id}>
+            {row.getVisibleCells().map(cell => (
+              <TableCellWithLink
+                key={cell.id}
+                linkProps={{
+                  to: '/tree/$treeId/test/$testId',
+                  params: {
+                    treeId,
+                    testId: row.original.id,
+                  },
+                  search: s => s,
+                }}
+              >
+                {flexRender(cell.column.columnDef.cell, cell.getContext())}
+              </TableCellWithLink>
+            ))}
+          </TableRow>
+        ))
+      : [
+          <TableRow key="no-results">
+            <TableCell colSpan={columns.length} className="h-24 text-center">
+              <FormattedMessage id="global.noResults" />
+            </TableCell>
+          </TableRow>,
+        ];
+  }, [modelRows, treeId]);
+
+  return (
+    <div className="flex flex-col gap-6 pb-4">
+      <TableStatusFilter filters={filters} onClickTest={onClickFilter} />
+      <PaginationInfo table={table} data={data} label="boots" />
+      <BaseTable headers={[]} headerComponents={tableHeaders}>
```

Comment:
make it that headers is undefined if you are using `headerComponents` 

you can achieve this by using `never`  in typescript

https://thomas-rubattel.medium.com/mutually-exclusive-props-in-typescript-703d7f0d74db

Reply:
Just realized that `headers` is not even being used anymore, you can just remove the prop

![image](https://github.com/user-attachments/assets/902e9753-94bb-4611-965a-b54278650797)